hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneAcl.java

```diff
@@ -340,4 +342,37 @@ public OzoneAcl setAclScope(AclScope scope) {
     ACCESS,
     DEFAULT;
   }
+
+  public String toAclString(AclEntry aclEntry) {
```

Comment:
Can we add documentation for these functions that String returned can be parsed using OzoneAcl#parseAcl?